services/dask-sidecar/.gitignore

```diff
@@ -0,0 +1 @@
+.dask-certificates
```

Comment:
Q: I wonder if in general we should assume that any service might have certificates and if so, they are in a pre-defined folder 

e.g. `services/${service-name}/docker/certs/*`    (in analogy to `/etc/ssl/certs` )

this way `*/certs/*` can always be ignored